app/models/solr_document.rb

```diff
@@ -24,6 +22,9 @@ class SolrDocument
   use_extension(Blacklight::Document::DublinCore)
 
   # Do content negotiation for AF models.
-
   use_extension(Hydra::ContentNegotiation)
+
+  def extent
```

Comment:
like this: https://github.com/projecthydra-labs/hyrax/blob/master/app/models/concerns/hyrax/solr_document/metadata.rb#L48